We do not need to check code format for this project.

--- expandvars-1.1.2/tox.ini.orig
+++ expandvars-1.1.2/tox.ini
@@ -7,5 +7,4 @@
     black
     pytest
 commands =
-    black --diff .
     pytest --cov --cov-report=html --cov-fail-under=100
--- expandvars-1.1.2/pyproject.toml.orig
+++ expandvars-1.1.2/pyproject.toml
@@ -29,7 +29,7 @@
 "Homepage" = "https://github.com/sayanarijit/expandvars"
 
 [project.optional-dependencies]
-tests = ['tox', 'pytest', 'pytest-cov', 'black']
+tests = ['tox', 'pytest', 'pytest-cov']
 
 [build-system]
 requires = ["hatchling"]
